### DISCUSSION TITLE
Option to select all values by default for multi variables

### DISCUSSION THREAD

**Discussion by LukasK13** [grafana/grafana#44387] [Dashboards, Panels & Visualizations]
As mentioned in #23677, an option, to select all values by default in a multi variable would be really helpful.
In my case, I have a multi variable that queries all available traces from a database which is defined by a second variable. The traces selected in the multi variable are displayed in a panel. If I change the selected database, I want Grafana to display all available traces by default and not only the traces that have been selected for the previous database and are still available in the currently selected database.

In #23677, the usage of a select regex field has been proposed which I think is a great idea. Unfortunately, the issue has been closed without proper discussion and is not being reopened.

**Comment by LukasK13** [comment]
I solved this problem by manually adding a select all button to grafana in my fork https://github.com/LukasK13/grafana/tree/8.4.1_selectAll

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
